HW06.Rmd

```diff
@@ -14,20 +15,47 @@ Steps:
   3. For 27 genes of interest AND your favorite gene, perform a t-test to see if it is differentially expressed between the WT vs H3K27M samples
   4. Create a graphing function and then create a boxplot that graphs expression between the two groups
   
-  
 #### Code
 
-From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data. 
-```{r load in data}
-#hint, using apply (specifically sapply) you can read in the data into a list and then bind the columns together. Or you can use a for loop too. 
+``` {r load packages, echo = FALSE, message = FALSE}
+library(readr)
+library(dplyr)
+library(knitr)
+library(tidyverse)
+```
+
+
+From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data.
+
+```{r load in data, message = FALSE}
+
+#Defining factors mainly for better control over the order of boxplots later on
+type <- factor(c("H3K27M_mutated", "WT"))
 
-#you only need the 
+#This chunk creates a list of .sf files in specified subfolder, batch loads them in, binds them together and applies "H3K27M_mutated" or "WT" factor to the values
+H3K27M_list <- Sys.glob("RNA_Seq_processed/H3K27M/*.sf")

```

Comment:
Cool--this function sys.glob() seems very handy in this case. (I didn't know about it!)